src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,130 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static final String VALID_USER_LOGIN = "login";
+    private static final String VALID_USER_PASSWORD = "password";
+    private static final int VALID_USER_AGE = 18;
+    private User validUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        validUser = new User(VALID_USER_LOGIN, VALID_USER_PASSWORD, VALID_USER_AGE);
+    }
+
+    @Test
+    void register_validUser_Ok() {
+        assertEquals(validUser,registrationService.register(validUser));
+        assertNotNull(validUser.getId());
+        assertTrue(Storage.people.contains(validUser));
```

Comment:
If I am not mistaken, we need to write test to the target functionality in code, that's why you could have one check to register validUser 
```suggestion

```